.buildkite/connected-tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+cp gradle.properties-example gradle.properties && cp -a example/properties-example/ example/properties/
```

Comment:
Since we are [already populating](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/trunk/.configure) the `example/properties` folder when we `applyConfiguration` in the next line, I think we can drop that part of the command.

```suggestion
cp gradle.properties-example gradle.properties
```


We should also update the memory in [gradle.properties-example](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/trunk/gradle.properties-example#L1) to something like `6G`, since we have a lot more memory in Buildkite machines than we did in CircleCI. Otherwise the builds will be unnecessarily slower and might even end up having out of memory issues.

P.S: I am opening a "suggestion PR" and will include these changes in that.